.github/actions/create-deployment-package/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Create deployment package"
```

Comment:
This action contains mostly what I copied from [dev tooling's example](https://github.com/coveo/deployment-pipeline/blob/master/.github/workflows/example-github-actions.yml).